concert/experiments/addons.py

```diff
@@ -1,16 +1,17 @@
 """Add-ons for acquisitions are standalone extensions which can be applied to them. They operate on
 the acquired data, e.g. write images to disk, do tomographic reconstruction etc.
 """
+import os
+import asyncio
 import logging
+import numpy as np
 from concert.coroutines.base import async_generate
 from concert.coroutines.sinks import Accumulate
 
-
 LOG = logging.getLogger(__name__)
 
 
 class Addon(object):
-
```

Comment:
trigger-happy pep8 fixer?

Reply:
Yes. I have a tool that removes the unnecessary empty lines and I run it over the full file. Should I leave it or put the unrelated ones back?

Reply:
yes please, let's have it clean.